### DISCUSSION TITLE
Add a Python website to the gallery ?

### DISCUSSION THREAD

**Discussion by linogaliana** [quarto-dev/quarto-cli#1123] [Show and Tell]
## Context

I create a complete course on `Python` for data scientists thanks to `quarto` (combined with `hugo`)  : https://pythonds.linogaliana.fr/ (github repo: https://github.com/linogaliana/python-datascientist). As you might see, the website is quite complex and also propose `folium` or `plotly` examples ([here](https://pythonds.linogaliana.fr/preprocessing/) or [here](https://pythonds.linogaliana.fr/cartotp/#des-cartes-r%C3%A9actives-gr%C3%A2ce-%C3%A0-folium) for instance)

I am really happy to finally have an equivalent to R Markdown for Python. Up to now, I was using hideous hacks to generate from the same `.Rmd` (using `reticulate`) both my static website and some notebooks. With `quarto`, this time is over and I now have consistent way to publish a static website and give students the possibility to start from the same content into notebooks as exercises. I propose several environments where they can open the notebook (e.g. google colab or binder)

## Question

I saw that you propose a really nice gallery of examples in https://quarto.org/docs/gallery/. Would it be possible to add my Python course website in this gallery ? This course website shows we can build very sophisticated websites with `quarto` and propose several outputs from the same source code.

**Comment by jjallaire** [comment]
Hi there! Thanks for letting us know about this. We're going to be expanding our gallery soon and will definitely add your site to it when we do.

**Reply by linogaliana** [reply]
Thanks, that would be great !

(and thanks again for `quarto`, that's a super powerful tool)
